File: chapter2/strings/string0.c
/**
 * Print each character of a string vertically
 */

 #include <cs50.h>
 #include <stdio.h>

 int main(void)
 {
    // Get a string from the user
    string s = get_string("Input: ");

    // Determine the length of the string
    // (how many letters before null character)
    int length = 0;
    while (s[length] != '\0')
    {
        length++;
    }

    // Loop through the string
    for (int i = 0; i < length; i++)
    {
        printf("%c\n", s[i]);
    }

 }

File: chapter5/more_python/string0.py
# Program that loops through each character in a string and
# prints it as a capital
from cs50 import get_string

# Get a string from user
s = get_string("Before: ")

print("After: ", end="")

# For each character in string
for c in s:
    # Print as uppercase
    print(c.upper(), end="")

print()
